CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 .. _fandango: https://github.com/tango-controls/fandango
 .. _79: https://github.com/ska-sa/tango-simlib/issues/79
 
+0.3.0
+-----
+- Support multiple device and control interfaces per server process (Enhancement)
+    - Addressing issue raised in #56
+- Adding a log when writing to the attributes
+- Reduced the unnecessary logging of the TANGO device
+- Handle non scalar value
+    - Addresses the  issue of not correctly reseting simulated quantity values to default
+    - When writing to a SPECTRUM type attribute, the data structure passed on to the write method is converted into an numpy          ndarray which was not handled. This was causing an error when it was trying to type cast an numpy.ndarray into an int.
```

Comment:
No need to add a long description here. Rather add a link to the issue/pull request. E.g. _Pull Request #209 (issue #207): Handle non scalar value_.

Reply:
how do you add a link

Reply:
@02Anele Take a look at line 2 and line 25.